### PR TITLE
Add in origin-kibana plugin with main.less precompile

### PR DESCRIPTION
**Pull request by jcantrill**
@sosiouxme @ewolinetz  @themurph  please review and comment.  I was unable to figure out how to install dependencies and compile the LESS file without install/uninstalling nodejs...booo

### CODE REVIEW

File: kibana/Dockerfile [hunk 1]
```diff
@@ -1,19 +1,33 @@
-FROM openshift/base-centos7
+FROM openshift/base-centos7 
 
 MAINTAINER Eric Wolinetz <ewolinet@redhat.com>
 
-ENV KIBANA_VER 4.1.1
+EXPOSE 5601
```
Comment (ewolinetz) [inline kibana/Dockerfile]: Do we want to expose port 5601 from the container?  Currently we let the template handle this and it maps it to port 80

Comment (sosiouxme) [inline kibana/Dockerfile]: What's in the docker file is really only used as guidance in forming the pod and service. The pod won't expose anything unless so configured.

File: kibana/kibana.yml [hunk 2]
```diff
@@ -19,8 +19,8 @@ kibana_index: ".kibana"
 # used by the Kibana server to perform maintence on the kibana_index at statup. Your Kibana
 # users will still need to authenticate with Elasticsearch (which is proxied thorugh
 # the Kibana server)
-kibana_elasticsearch_username: kibana
-kibana_elasticsearch_password: changeme
+#kibana_elasticsearch_username: kibana
```
Comment (ewolinetz) [inline kibana/kibana.yml]: Uncomment out these two lines please, at least until we change how we do auth throughout the logging stack

**Comment by ewolinetz** [comment]
@jcantrill one nit then LGTM

File: kibana/Dockerfile (hunk 1 above)
Comment (ewolinetz) [inline kibana/Dockerfile]: Ah, I misunderstood it's effect in the Dockerfile

**Pull request merged**